front-end/package.json

```diff
@@ -13,7 +13,7 @@
     "styled-components": "^5.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3001 react-scripts start",
```

Comment:
This starts up the Front End with port 3001 instead of 3000 (since the backend uses 3000)